packages/memory/package.json

```
@@ -0,0 +1,40 @@
+{
+	"name": "@directus/memory",
+	"version": "0.0.1",
```

Will be bumped by changesets (once the package is included in a changeset)
```suggestion
	"version": "0.0.0",
```